Classes/Eel/Helper/TrackingConfiguration.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Neos\GoogleAnalytics\Eel\Helper;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\ContentRepository\Domain\NodeAggregate\NodeName;
+use Neos\ContentRepository\Domain\Projection\Content\NodeInterface;
+use Neos\Eel\ProtectedContextAwareInterface;
+use Neos\Utility\Arrays;
+
+/**
+ * Class TrackingConfiguration
+ *
+ * @package Neos\GoogleAnalytics\Eel\Helper
+ */
+class TrackingConfiguration implements ProtectedContextAwareInterface
+{
+    /**
+     * @Flow\InjectConfiguration(path="default", package="Neos.GoogleAnalytics")
+     * @var array
+     */
+    protected $defaultSettings;
+
+    /**
+     * @Flow\InjectConfiguration(path="sites", package="Neos.GoogleAnalytics")
+     * @var array
+     */
+    protected $sitesSettings;
+
+
+    /**
+     * @param $site
+     * @param string|null $path
+     * @return mixed
+     */
+    public function setting($site, string $path = null)
```

Comment:
As site is optional, it would be better if it’s marked as such. And i think it can only be String or nodeinterface or null.